src/gc/gchandletable.cpp

```diff
@@ -22,3 +22,8 @@ void GCHandleTable::Shutdown()
 {
     Ref_Shutdown();
 }
+
+Object* GCHandleTable::ObjectFromHandle(OBJECTHANDLE handle)
+{
+    return OBJECTREFToObject(::HndFetchHandle(handle));
```

Comment:
yep, @swgillespie and I talked about what should be done wrt AD. aside from the alloc/survived bytes info that GC tracks for appdomain resource manager purpose, ideally everything else should goto the EE side.